### PR TITLE
Call BB focus from QT

### PR DESCRIPTION
**Pull request by SlimBenAmor**
Add javascript function to focus on basic block clicked from QT. In this PR, we:
 - determine and switch to the CFG of the function containing the clicked BB.
 - add a blink animation when focusing on a BB.

### CODE REVIEW

**Comment by SlimBenAmor** [comment]
@Alexandru-Tanase-Dev, @jordr can you please review and validate this PR ?

**Comment by jordr** [comment]
Should add this to the Kanban so we don't forget. IIRC @Alexandru-Tanase-Dev said there are some issues with this (doesn't work on first or second try, I forgot).

File: data/obviews/obviews.js [hunk 1]
```diff
@@ -180,8 +180,39 @@ function cfg_zoom(isZoomingIn=true, mouseEvt) {
...
+	else {
+		cfg_center_block_by_address(block_addr);
+	}
+}
+
+// center on a given block by its address of type int (Hex)
+function cfg_center_block_by_address(block_addr) {
 	const addrTitleRegex = new RegExp("(?<addr>[0-9a-fA-F]+):");
 	// browse all node to find the one with an address equal to the input
 	allNode = document.getElementsByClassName("node");
```
Comment (Alexandru-Tanase-Dev) [inline data/obviews/obviews.js:218]: Where allNode comes from? Should it be declared here?

Comment (Alexandru-Tanase-Dev) [reply data/obviews/obviews.js:218]: Line 227. Where is block variable declared?

Comment (SlimBenAmor) [reply data/obviews/obviews.js:218]: I will add `var` before them.

**Review by Alexandru-Tanase-Dev — APPROVED** [review]

**Pull request merged**